frontend/src/app/components/FeaturedExperiment/FeaturedButton.js

```diff
@@ -78,14 +77,17 @@ export default class FeaturedButton extends Component<FeaturedButtonProps, Featu
       this.sendMetric(ev, {eventLabel: "Popup Featured privacy"});
     };
 
-    return (<LocalizedHtml id={this.l10nId("legal-notice")}
-      $title={title}>
+    return (<Localized id={this.l10nId("legal_notice")}
```

Comment:
I had to change the id here from `legal-notice` to `legal_notice`. Even if there's some normalization going on in the `l10nId` util, the code which looks up a possible `_l10nsuffix` uses the id verbatim, which resulted in searching for `legal-notice_l10nsuffix`. It might be worth fixing the `experimentL10nId` util in a future PR.